src/components/pages/Main.vue

```diff
@@ -36,14 +40,11 @@ export default {
       console.log('Offers: ', response.data.offerIds);
 
       this.offersIds = response.data.offerIds;
-      this.closestOffer = this.offersIds[1];
-    }).then(async () => {
-      await console.log('Closest offer: ', this.offersIds[1]);
-
-      if (this.closestOffer !== this.offersIds[1]) {
-        console.log('Inside if');
-        this.closestOffer = this.offersIds[1];
+      for (let offerId = 0; offerId < this.offersIds.length; offerId += 2) {
```

Comment:
If number of offers is uneven, it tries to push undefined ID, resulting in 400 error. 

(console logging offerId and offerId + 1
![image](https://github.com/shareio-pl/shareio-frontend/assets/72970475/d4781868-69c3-40ac-8503-2ed93878dd48)